tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs

```diff
@@ -2076,6 +2077,7 @@ public void TestGroupWithFinalizeFunction()
         [Fact]
```

Comment:
Needs to be [SkippableFact]

Look for all of them. I won't add a comment to each one...